packages/ui-components/src/markdown/markdown.tsx

```diff
@@ -0,0 +1,293 @@
+import ReactMarkdown, { Options } from 'react-markdown';
+import remarkGfm from 'remark-gfm';
+
+import styled from '@darajs/styled-components';
+
+interface MarkdownProps extends Options {
```

Comment:
Should probably have standard `className` and `style` so it can be overriden outside?

Reply:
Good point, added